### DISCUSSION TITLE
HLS Streams in Public Pages

### DISCUSSION THREAD

**Discussion by YosefAdPro** [AzuraCast/AzuraCast#5464] [Q&A] [answered]
Hi,

Congratulations on adding HLS support.
Is it possible at this point to put HLS Streams in Public Pages?

many thanks.

**Comment by BusterNeece** [comment, marked as answer]
@YosefAdPro This is definitely on the agenda in the near future, but first we need to figure out a reliable way to track live listeners to the HLS stream. Once we've got that issue handled, we will be able to integrate the HLS stream into our main players.

**Comment by BusterNeece** [comment]
@YosefAdPro As of the currently deploying Rolling Release image (available in about 10 minutes), HLS support is now available on the public player, as we have worked out listener analytics behind the scenes.

**Reply by YosefAdPro** [reply]
Thanks for the update and for all your work!
